README.md

```diff
@@ -56,17 +56,26 @@ or download the [starter project](https://github.com/mcollovati/quarkus-hilla-st
-|  <picture><img alt="Maven Central 1.x" src="https://img.shields.io/maven-central/v/com.github.mcollovati/quarkus-hilla?style=for-the-badge&logo=apache-maven&versionPrefix=1" style="visibility: visible;"></picture>  | <picture><img alt="Quarkus 2.16+" src="https://img.shields.io/badge/QUARKUS-v2.16%2B-blue?style=for-the-badge&logo=Quarkus" style="visibility: visible;"></picture> | <picture><img alt="Hilla 1.3+" src="https://tinyurl.com/quarkus-hilla-h-1-3" style="visibility: visible;"></picture> | <picture><img alt="Flow 23.3+" src="https://img.shields.io/badge/VAADIN-v23.3%2B-blue?style=for-the-badge&logo=Vaadin" style="visibility: visible;"></picture> |
+|                                                                                                           Quarkus-Hilla                                                                                                            |                                                                               Quarkus                                                                               |                                                         Hilla                                                         |                                                                             Vaadin                                                                             |
+|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| <picture><img alt="Maven Central 2.4" src="https://img.shields.io/maven-central/v/com.github.mcollovati/quarkus-hilla?style=for-the-badge&logo=apache-maven&versionPrefix=2.4&color=green" style="visibility: visible;"></picture> |  <picture><img alt="Quarkus 3.1+" src="https://img.shields.io/badge/QUARKUS-v3.1%2B-blue?style=for-the-badge&logo=Quarkus" style="visibility: visible;"></picture>  | <picture><img alt="Hilla 2.4" src="https://tinyurl.com/quarkus-hilla-h-2-4-g" style="visibility: visible;"></picture> |   <picture><img alt="Flow 24.2" src="https://img.shields.io/badge/VAADIN-v24.2-blue?style=for-the-badge&logo=Vaadin" style="visibility: visible;"></picture>   |
```

Comment:
What is the intention to make the badge for quarkus-hilla and hilla green?
Also correct, that it is still vaadin 24.2?

Reply:
> What is the intention to make the badge for quarkus-hilla and hilla green?

I probably misunderstood your comment on Discord.

```
One idea for the compatibility matrix. Whats about changing the color of these versions, that changed compared to the previous version?
Example only for one tag, would also effect hilla, but quarkus stays sames, since not changed.
```
What do you think should be green in this PR?


> Also correct, that it is still vaadin 24.2?

Yes, Hilla 2.4 is based on Flow 24.2, since Vaadin 24.3 is not yet released